### PR TITLE
Feature/cleanup config file

### PR DESCRIPTION
**Pull request by frenchbread**
### Closes: 
- #939 
- #816 

### Changes
- Remove config files
- Fix feedback submit form

### CODE REVIEW

**Comment by brylie** [comment]
@frenchbread is this ready for review?

**Comment by frenchbread** [comment]
@brylie Yes. Please review.

File: server/methods/contactForm.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ Meteor.methods({
 
     // Send the e-mail
     Email.send({
-      to: Settings.findOne().contactForm_toEmail,
+      to: Settings.findOne().contactForm.toEmail,
```
Comment (brylie) [inline server/methods/contactForm.js]: What happens if this setting is not present? E.g. if the contact form 'to email' has not been provided in the settings collection? I suspect we will get an exception, which needs to be prevented or handled here.

Comment (frenchbread) [inline server/methods/contactForm.js]: Should we check if there is a value set in settings and if not, use email address from `settings.json`

**Pull request merged**